### ISSUE TITLE
Linter isn't warning me about lack of braces when --strict is set

### ISSUE THREAD

**Issue by GoogleCodeExporter** [google/closure-linter#39]
```
What steps will reproduce the problem?
1. wget 
http://closure-library.googlecode.com/svn-history/r1562/trunk/closure/goog/event
s/browserevent.js
2. curl http://codereview.appspot.com/download/issue5150047_3002_4002.diff | 
patch browserevent.js
3. gjslint --strict browserevent.js

What is the expected output? What do you see instead?

I expect a warning because of the lack of braces on the `if` statement starting 
on line 256. I don't see that warning.

What version of the product are you using? On what operating system?

Using Closure Linter r14.
```

Original issue reported on code.google.com by `amat...@gmail.com` on 7 Feb 2012 at 6:33
